src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -43,10 +44,45 @@ class Select<T> extends React.Component<Props<T>> {
 
     @observable selectedOptionRef: ?ElementRef<'li'>;
 
+    @observable buttonRefsByIndex: Map<number, ElementRef<'button'>> = new Map();
+
+    @observable searchText: string = '';
+
+    @observable focusedElementIndex: number = -1;
```

Comment:
do you think we need to store this inside of the component? as far as i see, the only time it is used is on line 242 where we could just request the focue without the check

Reply:
It's also used in line 222

Reply:
you are right, forgot about that 🙈